code/datums/abilities/flock/flockmind.dm

```diff
@@ -3,8 +3,29 @@
 ///////////////////////
 
 /datum/abilityHolder/flockmind
+	tabName = "Flockmind"
+	usesPoints = 1
+	points = 0 //total compute - used compute
+	var/tC = 0 //total compute
```

Comment:
Just call it total_compute, I know flock code is full of 1 or 2 letter vars but we can be better than that